3회차/오채현/1_소득불균형.py

```diff
@@ -1,3 +1,24 @@
 import sys
 
 sys.stdin = open("_소득불균형.txt")
+
+# testnum = 7
+# test = [15, 15, 15, 15, 15, 15, 15]
+T = int(input())
+
+for t in range(T):
+    n = int(input())
+    incomes = list(map(int, input().split()))
+
+    # total = sum(incomes)
+    total = 0
```

Comment:
`total` 값 설정에서 실수가 있었네요!
```suggestion
    total = sum(incomes)
    # total = 0
```